Update peer-review-overview.md with an image (#4546)
the-turing-way/the-turing-way: book/website/communication/peer-review/peer-review-overview.md

However, in current research practice, peer review often appears as a hurdle everyone needs to pass through.

> ```{figure} ../../../figures/peer-review-reply.*

Reviewer: ```suggestion
```{figure} http://www.phdcomics.com/comics/archive/phd050305s.gif
```

---

> height: 1154px

Reviewer: ```suggestion
```

  Correct Response: The reviewer raises and interesting concern. However, as the focus of this work is exploratory and not performance-based, validation was not foudn to be of critical importance to the contribution of the paper.
  Reviewer comment 2: The authors fail to reference the work of Smith et al who solved the same problem 20 years ago.
  How not to respond: Huh. We didn't think anybody had read that. Actually, their solution is better than ours.
  Correct response: The reviewer raises an interesting concern. However, our work is based on completely different first principles (we use different variable names), and has much more attractive graphical user interface.
  Reviewer comment 3: This paper is poorly written and scientifically unsound. I do not recommend it for publication.
  How not to respond: You #$% reviewer! I know who you are! I'm gonna get you when it is my turn to review!
  Correct response: The reviewer raises an interesting concern. However, we feel the reviewer did not fully comprehend the scope of the work, and misjudged the results based on incorrect assumptions. 
---

> How to respond to peer reviewers by [Piled Higher and Deeper](https://www.facebook.com/100051019341449/posts/pfbid02JVZRCChAwqbGQkSphNJTkn6FNHHvaVoiHm9STyj4nHBTaV8nY82L6pWCDxPZsmBkl/).

Reviewer: ```suggestion
[How to respond to peer reviewers by Jorge Cham, Copyright: Piled Higher and Deeper Publishing LLC](https://phdcomics.com/comics/archive.php?comicid=581).
```
